proto/tasks.proto

```diff
@@ -52,7 +52,7 @@ message TaskUpdate {
 // A task details update
 message TaskDetails {
     // The task's ID which the details belong to.
-    common.SpanId task_id = 1;
+    uint64 task_id = 1;
```

Comment:
i have a slight preference for continuing to have a semantically-named newtype for these, although i agree they shouldn't be `SpanId`s any longer. I would consider adding a `TaskId` message type in this file.

not a big deal though.